### DISCUSSION TITLE
Custom PK in Audit entity

### DISCUSSION THREAD

**Discussion by Rpry** [thepirat000/Audit.NET#420] [General]
Is it possible to set up custom PK in EF.Core version of your lib? By default it creates composit PK by Id and Datetime. Is it possible to change it?

**Comment by thepirat000** [comment]
You need to add details to your question. Can you point me to the default you're mentioning about the PK?

The EF data provider doesn't create the audit tables, so the audit table(s) structure is up to you. You only have to define a mapping between the entities to be audited and the audit tables.

**Reply by Rpry** [reply]
Thanks, problem is resolved
